layouts/partials/scripts.html

```diff
@@ -1,3 +1,5 @@
-<script src="{{ .Site.BaseURL }}assets/scripts/main.js"></script>
+<!-- Uncomment once we have some actual code to include
+    <script src="{{ .Site.BaseURL }}assets/scripts/main.js"></script>
+-->
```

Comment:
I'm wary of deleting the whole gulp process (and I'm not deleting `start.js`, just emptying it - so it's there as an obvious place to put code). But I can't fully justify _not_ deleting it all. And you're right, if we don't delete it, then adding some comments would be good.